lib/app-tray.js

```diff
@@ -45,10 +42,20 @@ exports.init = function (gme) {
     type: 'separator'
   }));
   trayMenu.append(new MenuItem({
+    label: 'Show / Hide Google Music Electron',
+    click: gme.onTrayClick
+  }));
```

Comment:
I should add, at least on Mac OS X, that the menu bar icon offers no indication of which application it is, and in case you're not really familiar with the icon, it would be nice to have it say "Google Music Electron" in the menu, instead of just "window"